source/updateCheck.py

```diff
@@ -102,6 +103,7 @@ def checkForUpdate(auto=False):
 		"autoCheck": auto,
 		"allowUsageStats":allowUsageStats,
 		"version": versionInfo.version,
+		"buildVersion":buildVersion.formatBuildVersionString(),

```

Comment:
you don't necessarily have to import buildVersion, as it is already imported by versionInfo.

```suggestion
		"buildVersion":versionInfo.formatBuildVersionString(),
```

Having said that, as versionInfo is the file with all the localized data, it is probably better to go for buildVersion and leave out versionInfo from the updateCheck module. Anyway, I'd go for either one or the other.